resources/virt.go

```diff
@@ -0,0 +1,521 @@
+package resources
+
+import (
+	"encoding/gob"
+	"log"
+	"fmt"
+	"strconv"
+
+	"github.com/purpleidea/mgmt/event"
+	"github.com/rgbkrk/libvirt-go"
+	"math/rand"
+	"time"
+	"bytes"
+)
+
+func init() {
+	gob.Register(&VirtRes{})
+}
+
+type VirtRes struct {
+	BaseRes `yaml:",inline"`
+	Cpus int `yaml:"cpus"`
+	Ram int `yaml:"ram"`
```

Comment:
Please add a comment here at the end to specify the units. Shouldn't it also be uint?